File: eval/fixtures/algorithms/pair-bug.py
def solve(a, target):
    left, right = 0, len(a) - 1
    while left < right:
        value = a[left] + a[right]
        if value == target:
            return True
        if value < target:
            right -= 1
        else:
            left += 1
    return False

File: eval/fixtures/algorithms/pair-bug.js
function solve(a, target) {
  let left = 0, right = a.length - 1;
  while (left < right) {
    const value = a[left] + a[right];
    if (value === target) return true;
    if (value < target) right--;
    else left++;
  }
  return false;
}
